gulpfile.js

```diff
@@ -41,21 +34,25 @@ gulp.task('lint', function() {
   // Note: To have the process exit with an error code (1) on lint error, return
   // the stream and pipe to failOnError last.
   return gulp.src([
-      'app/**/*.{js,jsx}',
-      'tests/**/*.js',
-      '!app/components/**',
-      '!app/hookSW.js'
+      APP_ROOT + '**/*.{js,jsx}',
+      TESTS_ROOT + '**/*.js',
+      './gulpfile.js',
+      './karma.conf.js',
+      '!' + APP_ROOT + 'components/**',
     ])
     .pipe(eslint())
     .pipe(eslint.format())
     .pipe(eslint.failAfterError());
 });
 
 /**
- * copies necessary files for the Babel amd loader to the app.
+ * Copies necessary files for the Babel amd loader to the app.
  */
 gulp.task('loader-polyfill', function() {
-  return gulp.src(['./node_modules/alameda/alameda.js', 'app/js/bootstrap.js'])
+  return gulp.src([
+      './node_modules/alameda/alameda.js',
+      APP_ROOT + 'js/bootstrap.js',
```

Comment:
__note:__ it will likely fail with your latest ESLint PR :) It will likely require template string - so make sure land them rebased and with green travis :)